run/GCHP/setCommonRunSettings.sh.template

```diff
@@ -245,6 +269,13 @@ fi
 # Mass tuning factor is used in the HEMCO DustDead extenstion for GEOS-Chem
 # benchmarking and is resolution-dependent. We recommend using offline dust
 # instead of the online extension for GCHP science runs.
+#
+# NOTE: We have retuned the dust scaling factors to account for the
```

Comment:
Why does this note appear again?

Reply:
@lizziel: We have simplified the logic where mass tuning factors are set in commit d08e541762be53774d818deaf691a59df4890c47.